packages/design-system-tokens/src/lib/exportCssVars.ts

```diff
@@ -133,15 +132,29 @@ const writeCssFile = (filename: string, importedModule: any, sep: string) => {
  * @returns An exit code based on success writing output
  */
 export const exportCssVars = (fileDescriptors: FileDescriptor[], outPath: string): number => {
-  fileDescriptors.forEach((file) => {
-    const importedModule = require(`${file.moduleImportName}`);
-    // component files do not need a separator
-    const sep = file.baseName.includes('component') ? '' : '-';
-
-    const layoutFilename = `${outPath}/${file.baseName}-layout-tokens.scss`;
-    const cssVarFilename = `${outPath}/${file.baseName}-theme.css`;
-    writeSassLayout(layoutFilename, file, importedModule, sep);
-    writeCssFile(cssVarFilename, importedModule, sep);
+  const groupByTheme = fileDescriptors.reduce((acc: Record<string, any[]>, file) => {
+    const theme = file.baseName.split('-')[0];
+    if (!acc[theme]) {
+      acc[theme] = [];
+    }
+    acc[theme].push(file);
+    return acc;
+  }, {});
+
+  Object.entries(groupByTheme).forEach(([theme, files]) => {
+    let output = '';
+
+    files.reverse().forEach((file) => {
+      const importedModule = require(`${file.moduleImportName}`);
+      // component files do not need a separator
+      const sep = file.baseName.includes('component') ? '' : '-';
+      const layoutFilename = `${outPath}/${theme}-layout-tokens.scss`;
+      writeSassLayout(layoutFilename, file, importedModule, sep);
```

Comment:
It looks like this part can move down to line 156, because I think it's actually writing the file twice! This would also be a good opportunity to add some more comments about why we need to write this file in the first place.